pkgs/development/python-modules/jsonstreams/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytest, six, }:
+
+buildPythonPackage rec {
+  pname = "jsonstreams";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "dcbaker";
+    repo = pname;
+    rev = version;
+    sha256 = "0c85fdqkj5k4b0v0ngx2d9qbmzdsvglh4j9k9h7508bvn7l8fa4b";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  checkInputs = [ pytest ];
```

Comment:
Is it not possible to use pytestCheckHook?

Reply:
Thank you for showing me that this exists.  Now with pytestCheckHook.